template/registry.go

```diff
@@ -61,6 +62,15 @@ func (o Options) GetInt(name string, fallback int, validate ...Validate[int]) (i
 		return fallback, nil
 	}
 
+	if s, ok := v.(string); ok {
+		var err error
+
+		v, err = strconv.ParseInt(s, 10, 32)
+		if err != nil {
+			return 0, fmt.Errorf("parse integer from string '%s': %w", s, err)
```

Comment:
We use double quotes in error messages for string values. ALWAYS. No single quotes.